cypress/support/commands.js

```diff
@@ -0,0 +1,149 @@
+import * as testData from '../fixtures/testData.json'
+import Selectors from "../support/selectors.js"
+
+Cypress.Commands.add('checkMainPageIsOpen', () => {
+    cy.url().should('eq', testData.carsMainPage)
+    cy.get(Selectors.logo).should('be.visible')
+})
+
+Cypress.Commands.add('goToResearchReviewsPage', () => {
+    cy.get(Selectors.pageMenuTitles).contains(testData.testResearchLinkTitle).click()
+})
+
+Cypress.Commands.add('checkResearchReviewsPageIsOpen', () => {
+    cy.url().should('contain', testData.researchPage)
+    cy.get(Selectors.researchHeading).contains(testData.testResearchLinkTitle)
+})
+
+Cypress.Commands.add('selectRandomOption', (optionSelector, menuSelector) => {
```

Comment:
Three things here:
1. There are hardcodes like `+ 1` or `0`, `- 1`. Better to delegate them to the test data variable with a descriptive name
2. Function for getting random integer would be perfect to put to some kind of Utils file.
3. Implementation looks complex to me. I believe this would do the same `return cy.get(optionSelector).select(getRandomInt()).invoke('text')`